modules/op-installing-results.adoc

```diff
@@ -0,0 +1,64 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-chains-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="installing-results_{context}"]
+= Installing {tekton-results}
+
+[role="_abstract"]
+To install {tekton-results}, you must provide the required resources and then create and apply a `TektonResult` custom resource (CR). The {pipelines-shortname} Operator installs the Results services when you apply the `TektonResult` custom resource.
+
+.Procedure
+
+. Create a secret with the database user named `result` and a random password in the `openshift-pipelines` namespace by entering the following command:
++
+[source,bash]
+----
+oc create secret generic tekton-results-postgres --namespace=openshift-pipelines --from-literal=POSTGRES_USER=result --from-literal=POSTGRES_PASSWORD=$(openssl rand -base64 20)
+----
+
+. Generate a new self-signed certificate by entering the following command:
++
+[source,bash]
+----
+openssl req -x509 \
+-newkey rsa:4096 \
+-keyout key.pem \
+-out cert.pem \
+-days 365 \
+-nodes \
+-subj "/CN=tekton-results-api-service.${NAMESPACE}.svc.cluster.local" \
+-addext "subjectAltName = DNS:tekton-results-api-service.${NAMESPACE}.svc.cluster.local"
+----
++
+[NOTE]
+====
+If you have an existing certificate provided by a certificate authority (CA), you can use this certificate instead of generating a self-signed certificate.
```

Comment:
```suggestion
If you have an existing certificate provided by a certificate authority (CA), use this certificate instead of generating a self-signed certificate.
```